x-pack/plugins/security_solution/public/resolver/view/panels/panel_content_process_detail.tsx

```diff
@@ -0,0 +1,157 @@
+  processEvent,
+  pushToQueryParams,
+}: {
+  processEvent: ResolverEvent;
+  pushToQueryParams: (arg0: CrumbInfo) => unknown;
+}) {
+  const processName = processEvent && event.eventName(processEvent);
+  const processInfoEntry = useMemo(() => {
+    let dateTime = '';
+    const eventTime = processEvent && event.eventTimestamp(processEvent);
+    if (eventTime) {
+      dateTime = formatDate(eventTime);
+    }
+
+    const processInfo = processEvent
+      ? {
+          [i18n.translate('xpack.securitySolution.enpoint.resolver.panel.processDescList.created', {
+            defaultMessage: 'Created',
+          })]: dateTime,
+          [i18n.translate('xpack.securitySolution.enpoint.resolver.panel.processDescList.path', {
+            defaultMessage: 'Path',
+          })]: processPath(processEvent),
+          [i18n.translate('xpack.securitySolution.enpoint.resolver.panel.processDescList.pid', {
+            defaultMessage: 'PID',
+          })]: processPid(processEvent),
+          [i18n.translate('xpack.securitySolution.enpoint.resolver.panel.processDescList.user', {
+            defaultMessage: 'User',
+          })]: (userInfoForProcess(processEvent) as { name: string; domain: string }).name,
+          [i18n.translate('xpack.securitySolution.enpoint.resolver.panel.processDescList.domain', {
+            defaultMessage: 'Domain',
+            [i18n.translate(
+              'xpack.securitySolution.enpoint.resolver.panel.processDescList.parentPid',
+              {
+                defaultMessage: 'Parent PID',
+              }
+            )]: processParentPid(processEvent),
+          })]: (userInfoForProcess(processEvent) as { name: string; domain: string }).domain,
+          [i18n.translate('xpack.securitySolution.enpoint.resolver.panel.processDescList.md5hash', {
+            defaultMessage: 'MD5',
+          })]: md5HashForProcess(processEvent),
+          [i18n.translate(
+            'xpack.securitySolution.enpoint.resolver.panel.processDescList.commandLine',
+            {
+              defaultMessage: 'Command Line',
+            }
+          )]: argsForProcess(processEvent),
+        }
+      : {};
+
+    return Object.entries(processInfo)
+      .filter(([, description]) => {
+        return description;
+      })
+      .map(([title, description]) => {
+        return { title, description: String(description) };
+      });
+  }, [processEvent]);
+
+  const crumbs = useMemo(() => {
+    return [
+      {
+        text: i18n.translate(
+          'xpack.securitySolution.enpoint.resolver.panel.processDescList.events',
+          {
+            defaultMessage: 'Events',
+          }
+        ),
+        onClick: () => {
+          pushToQueryParams({ crumbId: '', crumbEvent: '' });
+        },
+      },
+      {
+        text:
+          i18n.translate('xpack.securitySolution.enpoint.resolver.panel.processDescList.details', {
+            defaultMessage: 'Details for: ',

```

Comment:
you'll need to pass the `processName` into the formatting function and use a string like `'Details for: {processName}'` to allow translators to place the process name wherever in the string.

Reply:
https://github.com/elastic/kibana/pull/69098/commits/efb24f1604b5d81851776cb55f4fca514569d62d

Reply:
Thanks for the spot.